common/build.gradle.kts

```diff
@@ -53,6 +53,9 @@ dependencies {
     implementation(libs.iconics.core)
     implementation(libs.community.material.typeface)
 
+    implementation(libs.compose.material3)
+    api(libs.kotlinx.serialization.json)
```

Comment:
We actually already have `libs.kotlinx.serialization.json` as a dependency in the plugin file, so my suggestion would be adjusting it there instead: https://github.com/home-assistant/android/blob/fb021d86cf46c28bbe65037f49a69ea804a459ca/build-logic/convention/src/main/kotlin/AndroidCommonConventionPlugin.kt#L110